core/src/gl/framebuffer.cpp

```diff
@@ -147,7 +147,7 @@ void FrameBuffer::init(RenderState& _rs) {
     }
 
     GLenum status = GL::checkFramebufferStatus(GL_FRAMEBUFFER);
-    GL_CHECK();
+    GL_CHECK({});
```

Comment:
Why exactly is that change needed?

Reply:
MSVC does not support GL_CHECK with empty argument it does not resolve it to empty string, that's why you need to provide C++ statement - see macro

Reply:
Done